### PR TITLE
Frontend setup

### PR DESCRIPTION
**Pull request by azboss2021**
**_WHAT DOES THIS PR DO?_**
This PR sets up the frontend directory, and adds some basic libraries needed to style the UI of the application.

**_WHAT ISSUES ARE RELATED TO THIS PR?_**
- #17 

**_HOW DO I TEST OUT THIS PR?_**
- Go into the frontend directory and run `npm run dev` and open http://127.0.0.1:5173/ in your browser. It will give you a link in the console as well.

### CODE REVIEW

**Review by chrisj117 — APPROVED** [review]
As discussed in the meeting, this PR looks good.

**Comment by danielogen** [comment]
Nice @azboss2021, lets move the `frontend` content to `web` directory

File: frontend/README.md
Comment (danielogen) [inline frontend/README.md:1]: I guess we can remove this README since we have one at the root directory

**Review by tresimovic — APPROVED** [review]
As discussed in the team meeting, this PR looks good.

**Review by danielogen — APPROVED** [review]
All reviews (issues) have been resolved. This looks good now

**Pull request merged**